app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+class CommandError(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+
+    if len(command_list) != 3:
+        return
```

Comment:
Instead of returning silently when the command list does not have exactly 3 elements, consider raising a `CommandError` with a descriptive message. This will help in debugging and understanding what went wrong.